haiku-apps/functionplotter/functionplotter-0.9.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Plots an abitrary number of functions in parameter form"
+DESCRIPTION="The graph can be saved to a file as a bitmap, can be printed out \
+to a printer and can be copied to the clipboard.
+
+Includes a small functional programming language to define constants and \
+functions."
+HOMEPAGE="https://github.com/haikuports/haikuports/"
+COPYRIGHT="2000 Michael Pfeiffer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/haikuports/haikuports/files/2381969/FunctionPlotter0.9.2.src.zip"
+CHECKSUM_SHA256="07795a291a3c756e89a1d49d43ea2d2628e9728d38c8aaeb96ee1d70e7fae548"
+SOURCE_DIR="FunctionPlotter0.9.2"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	functionplotter = $portVersion
+	app:FunctionPlotter = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:xres
+	"
+
+BUILD()
+{
+	cp $portDir/additional-files/Makefile ./src/Makefile
+	make $jobArgs -C src
+}
+
+INSTALL()
+{
+	mkdir -p "$appsDir"
+	cp ./src/FunctionPlotter "$appsDir"
+
+	addAppDeskbarSymlink $appsDir/FunctionPlotter
```

Comment:
Suggestion:
```diff
-	addAppDeskbarSymlink $appsDir/FunctionPlotter
+	addAppDeskbarSymlink "$appsDir"/FunctionPlotter
```
(This would be consistent with the quotes we have in`"$appsDir"` in the previous lines.)

Reply:
Bugger, missed that one ;)